corehq/apps/app_manager/views/releases.py

```diff
@@ -101,11 +101,12 @@ def paginate_releases(request, domain, app_id):
     results = app_es.exclude_source().run()
     app_ids = results.doc_ids
     apps = get_docs(Application.get_db(), app_ids)
-    for app in apps:
-        app.pop('translations')
     saved_apps = [SavedAppBuild.wrap(app, scrap_old_conventions=False).to_saved_build_json(timezone)
                   for app in apps]
 
+    for saved_app in saved_apps:
+        saved_app.pop('multimedia_map')
+        saved_app.pop('media_language_map')
```

Comment:
@dannyroberts is it safe to add these two in `to_saved_build_json` itself when ignoring fields like `translations`.
The method `to_saved_build_json` is used at one more place so was not sure if it was safe to ignore these two in the method itself

Reply:
I don't know the answer to your question, but I like where your head's at.

Reply:
The other place you're referring to is the `save_copy` view, right? It just sends the `to_saved_build_json` output as part of its JSON response to presumably to the same releases page so that the new release can be added to the top of the list of releases in the UI. If that's the case, I'd say it seems right to remove these fields a level up inside of `to_saved_build_json` like you're suggesting.

Reply:
If that does end up being the case, you could change the method name or add a docstring to the method making it clear that it's used explicitly for the releases page.  Could also change it to a utility function rather than a model method if it's not intended to be generically useful.

Reply:
Thanks Both.
Did the rename and documentation. Was not comfortable with moving it as utility function, since that would have needed to touch couple of more places, and tests.